manage.py

```diff
@@ -22,6 +22,7 @@ def make_shell_context():
 
 manager.add_command('shell', Shell(make_context=make_shell_context))
 manager.add_command('db', MigrateCommand)
+manager.add_command('runserver', Server(host="0.0.0.0"))
```

Comment:
I feel that runserver is already a command baked into manage.py. Can you explain if we need to keep it in here?
Also, I can't seem to see why `import os` is needed.

Reply:
Yes, runserver is already the command backed into manage.py, but by default, the host variable of the `Server()` contructor is `'127.0.0.1'`. The problem is that for we can access flask app inside the docker container, the host should be `'0.0.0.0'`, otherwise it wont work. So that's the reason why we have to keep it.
I have deleted `import os` because is no longer needed.